ServiceBased/ProAddIn/WorkflowManagerSampleAddIn/GetJobInformation.cs

```diff
@@ -0,0 +1,49 @@
+﻿using ArcGIS.Core.Data;
+using ArcGIS.Core.Geometry;
+using ArcGIS.Desktop.Catalog;
+using ArcGIS.Desktop.Core;
+using ArcGIS.Desktop.Editing;
+using ArcGIS.Desktop.Extensions;
+using ArcGIS.Desktop.Framework;
+using ArcGIS.Desktop.Framework.Contracts;
+using ArcGIS.Desktop.Framework.Dialogs;
+using ArcGIS.Desktop.Framework.Threading.Tasks;
+using ArcGIS.Desktop.Layouts;
+using ArcGIS.Desktop.Mapping;
+using ArcGIS.Desktop.Workflow.Client;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WorkflowManagerSampleAddIn
+{
+    internal class GetJobInformation : Button
+    {
+        protected override void OnClick()
+        {
+
+            // Get the Workflow Manager server url
+            var jobsManager = WorkflowClientModule.JobsManager;
+            var jobId = "HSoSXIeFSmu3nuV4rrEzsA";
```

Comment:
Should the Readme reference these hard-coded jobIds so the user realizes they are here?

Reply:
Added info in the README and updated some code comments.